SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -0,0 +1,326 @@
+====
+
+[#ref_tbl_pid8_mapping]
+.PID-8 Administrative Sex Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|PID-8/IS-1
+|Administrative Sex
+|pm:PatientContextState
+/pm:CoreData
+/pm:Sex
+|Note that the HL7 Administrative Sex value set (HL7 table 0001) differs from the SDC pm:Sex value set and requires a mapping accordingly (see also <<ref_tbl_sex_mapping>>).
+
+|===
+
+[#ref_tbl_sex_mapping]
+.Patient's Sex Value Set Mapping
+|===
+|SDC Value |SDC Description |HL7 Value |HL7 Description
+
+|Unspec
+|Unspecified. Sex is not designated.
+|A
+|Ambiguous
+
+|M
+|Male. Indicates a male patient.
+|M
+|Male
+
+|F
+|Female. Indicates a female patient.
+|F
+|Female
+
+|Unkn
+|Unknown. Indicates that the sex is unknown for different reasons.
+|U
+|Unknown
+
+|===
+
+// https://build.fhir.org/ig/HL7/uv-pocd/StructureDefinition-Patient.html
+
+====== PID-10 Race
+The PID-10 field shall contain the patient's race. The table <<ref_tbl_pid10_mapping>> defines the mapping of the SDC patient's race information to the data fields of the HL7 data type *CWE* used in the PID-10 field.
+
+[#ref_tbl_pid10_mapping]
+.PID-10 Race Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|PID-10/CWE-1
+|Identifier
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@Code
+|
+
+|PID-10/CWE-2
+|Text
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@SymbolicCodeName
+|
+
+|PID-10/CWE-3
+|Name of Coding System
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@CodingSystem
+|
+
+|PID-10/CWE-4
+|Alternate Identifier
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@Code
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|PID-10/CWE-6
+|Name of Alternate Coding System
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@CodingSystem
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|PID-10/CWE-7
+|Coding System Version ID
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@CodingSystemVersion
+|
+
+|PID-10/CWE-8
+|Alternate Coding System Version ID
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@CodingSystemVersion
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|===
+
+====== PID-31 Identity Unknown Indicator
+The PID-31 field shall indicate whether or not the patient's identity is known.

```

Comment:
I think this is rather a definition:
```suggestion
The PID-31 field indicates whether or not the patient's identity is known.
```

Reply:
Agree with the English Grammar Police: Simplify to only "whether".  That said, I hear people use both every day so ... it's not like anyone (but The Grammar Police!!!) would ever even notice, let alone say anything.

Reply:
Done.